### ISSUE TITLE
[FEAT] Specviz: add possibility to load fits files with units e/s for the flux

### ISSUE THREAD

**Issue by camipacifici** [spacetelescope/jdaviz#753]
### Description
 
I would like to be able to load in Specviz a fits file with the flux in e/s. Currently, it seems like the additional units are only u.Unit('adu') and u.Unit('ct/s')

**Comment by pllim** [comment]
@rosteen , do you mean adding `'electron/s'` here?

https://github.com/astropy/specutils/blob/6dd54ad3bd75b52066f8fc5c7661591eb8581af9/specutils/io/parsing_utils.py#L211

**Comment by rosteen** [comment]
Yes, but there may be a couple other things to do as well, I'm not sure yet.
